keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -591,6 +588,18 @@ service Provisioner {
 This call is meant to retrieve the unique provisioner Identity. This identity will have to be set in `BucketRequest.Provisioner` field in order to invoke this specific provisioner.
 
 ```
+message Protocol {
+	// ProtocolName is the name of the protocol
```

Comment:
Do you have an example protocol name? (The previous code said "one of s3, gs, azureBlob").

Is that really about *protocol* though? That is does "S3" mean the "S3 XML API" (which many providers / things speak to varying degrees, including MinIO and Ceph).

Reply:
From the kep:
```
protocol: protocol-specific field the application will use to access the backend storage.
   name: supported protocols are: "s3", "gs", and "azureBlob".
   version: version being used by the backend storage.
   azureBlob: data required to target a provisioned azure container.
   gs: data required to target a provisioned GS bucket.
   s3: data required to target a provisioned S3 bucket.
```
Perhaps @wlan can provide an example?